src/ceph-volume/ceph_volume/devices/lvm/strategies/bluestore.py

```diff
@@ -103,8 +103,7 @@ def execute(self):
 
         # create the lvs from the vgs captured in the beginning
         for create in osd_vgs.values():
-            block_uuid = system.generate_uuid()
-            lvs = lvm.create_lvs('osd-data', block_uuid, vg=create['vg'], parts=create['parts'])
+            lvs = lvm.create_lvs(create['vg'], parts=create['parts'], name_prefix='osd-data')
```

Comment:
in theory type checkers should be able to find these errors without any type hints. Unfortunately mypy is not there yet.